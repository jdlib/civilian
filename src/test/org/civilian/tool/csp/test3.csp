encoding ISO-8859-1
package com.test;

import java.io.Serializable
import java.io.IOException
import ./data.Data1
import ../Data2;

template
	package-access
	extends - using TemplateWriter
	implements Serializable
	throws IOException
{{
	Hello!
}}
