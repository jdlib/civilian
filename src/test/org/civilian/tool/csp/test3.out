/**
 * Generated from test3.csp
 * Do not edit.
 */
package com.test;


import java.io.IOException;
import java.io.Serializable;
import com.Data2;
import com.test.data.Data1;
import org.civilian.template.TemplateWriter;


class Test3 implements Serializable
{
	public synchronized void print(TemplateWriter out) throws IOException
	{
		if (out == null)
			throw new IllegalArgumentException("out is null");
		this.out = out;
		print();
	}


	protected void print() throws IOException
	{
		out.println("Hello!");                                          // line 15: Hello!
	}


	protected TemplateWriter out;
}
